client/package.json

```diff
@@ -335,5 +335,9 @@
         "never"
       ]
     }
+  },
+  "prettier": {
+    "arrowParens": "avoid",
```

Comment:
Do we still need 
```
--no-semi --arrow-parens=avoid
``` 
in  

```
"lint-fix": "prettier --write --no-semi --arrow-parens=avoid '*.js' 'config/**/*.js' 'platform/**/*.js' 'scripts/**/*.js' 'src/**/*.js' 'stories/**/*.js' 'tests/**/*.js' && eslint --fix '*.js' 'config/**/*.js' 'platform/**/*.js' 'scripts/**/*.js' 'src/**/*.js' 'stories/**/*.js' 'tests/**/*.js'",
```
?

Reply:
Prettier's defaults are `semi: true` and `arrow-parens: always`. I personally don't like that. And changing them (esp. semi) will result in a huge commit.

Reply:
@gjvoosten This change doesn't affect the prettier configuration because we are not changing the rules. It basically moves configuration to package.json from command line arguments. I am using WebStorm for development and this is the only way I could find in order to make prettier, eslint and WebStorm play along nicely when formatting. But I am wondering if this change affects your IDEs code formatting configuration?

@VassilIordanov No, we do not need command line arguments anymore with this change.